pandas/core/ops/__init__.py

```diff
@@ -624,7 +629,7 @@ def to_series(right):
 
 
 def _should_reindex_frame_op(
-    left: "DataFrame", right, op, axis, default_axis: int, fill_value, level
+    left: "DataFrame", right, op, axis, default_axis, fill_value, level
```

Comment:
why is this changed?

Reply:
without this i got mypy complaints that i havent been able to track down

Reply:
Can you share the error message?

Reply:
`pandas/core/ops/__init__.py:706: error: Argument 5 to "_should_reindex_frame_op" has incompatible type "Optional[str]"; expected "int"`

Reply:
Gotcha. Looks like `_get_frame_op_default_axis` returns an Optional string and that is used as the default argument in the `f` function local to `_arith_method_FRAME`.

So makes sense Mypy would complain about that. Whether the existing annotation is wrong or if its discovering a bug for us I'm not sure

Reply:
i think its the annotation.  there is some more code that can be ripped out after this, so im planning to do an annotation pass towards the end